piker/_daemon.py

```diff
@@ -145,13 +160,28 @@ async def spawn_brokerd(
     global _services
     assert _services
 
-    await _services.actor_n.start_actor(
+    portal = await _services.actor_n.start_actor(
         dname,
         enable_modules=_data_mods + [brokermod.__name__],
         loglevel=loglevel,
         **tractor_kwargs
     )
 
+    # TODO: so i think this is the perfect use case for supporting
```

Comment:
Link with goodboy/tractor#53, pretty sure the context entry / exit sync stuff would work amazing for this.